packages/dds/tree/src/test/simple-tree/proxies.spec.ts

```diff
@@ -117,39 +117,6 @@ describe("SharedTreeObject", () => {
 		assert.equal(root.polyChild.content, "42");
 	});
 
-	it("can narrow polymorphic value fields", () => {
```

Comment:
There are a few problems with these tests:

1. They don't all test narrowing: since assert.equal isn't strict about what types it accepts.
2. They are testing Tree.is, not the proxies, so this file is the wrong place for them
3. They don't test that is returns the correct result (each of these ifs should have a fail in one of the cases)
4. The first tests doesn't even read the same value (polyChild)  it is branching on (polyValue)

Thus these tests have been adjusted and moved.